components/var_dumper.rst

```diff
@@ -875,3 +877,86 @@ that holds a file name or a URL, you can wrap them in a ``LinkStub`` to tell
+other named arguments you may use to display your own variables. For example, if
+you want to display a debug stacktrace with your variable content, you can do it
+as follow::
+
+    $var = ['test'];
+    dump($var, _trace: true);
+
+Here is the list of the supported options:
+
+* ``_format``: explicitly choose the format to display the output between ``cli``,
+  ``html`` or ``server``.
+* ``_trace``: displays the debug backtrace from where ``dump()`` has been called.
+  This option accepts a boolean (``true`` to display the trace, ``false`` to hide it),
+  but also an integer. This integer limits the stacktrace size being displayed.
+* ``_theme``: set the theme to use when using ``html`` format, between ``dark`` or ``light``.
+* ``_charset``: set the charset to use for the output content.
+* ``_flags``: a bitmask of ``AbstractDumper::DUMP_*`` flags. To see available flags, have a
+  look at :class:`Symfony\\Component\\VarDumper\\Dumper\\AbstractDumper`.
+* ``_max_items``: set the value being passed to
+  :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxItems` of the default
+  cloner. For more information, see the :ref:`Cloners <var-dumper-cloners>` section.
+* ``_min_depth``: set the value being passed to
+  :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMinDepth` of the default
+  cloner. For more information, see the :ref:`Cloners <var-dumper-cloners>` section.
+* ``_max_string``: set the value being passed to
+  :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxString` of the default
+  cloner. For more information, see the :ref:`Cloners <var-dumper-cloners>` section.
+* ``_max_depth``: set the value being passed to
+  :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxDepth` of the default
+  cloner. For more information, see the :ref:`Cloners <var-dumper-cloners>` section.
+* ``_max_items_per_depth``: set the value being passed to
+  :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxItemsPerDepth` of the default
+  cloner. For more information, see the :ref:`Cloners <var-dumper-cloners>` section.
+
+You can also set these options globally thanks to the ``VAR_DUMPER_OPTIONS`` environment variable,
+formatted as a query string:
+
+.. code-block:: bash
+
+    # .env (or .env.local)
+    VAR_DUMPER_OPTIONS="_trace=3&_max_items=4&_theme=light"
+
+Finally, the :class:`Symfony\\Component\\VarDumper\\Dumper\\VarDumperOptions` is an option
+builder that you can use to create your options array thanks to a fluent interface. This
+class provides the ``VarDumperOptions::toArray()`` method to get the options as an array
+and use them in a ``dump()`` call with to the spread operator.
+
+.. note::
+
+    If a format is set with ``VAR_DUMPER_FORMAT`` environment variable, it will be used instead
+    of the one defined by the ``VAR_DUMPER_OPTIONS`` environment variable.
+
+.. versionadded:: 6.3
+
+    Special options of ``dump()`` were introduced in Symfony 6.3.
```

Comment:
"Special options" is not clear to me. Do you have `VAR_DUMPER_OPTIONS` in mind ?